examples/orientation_utils_planar.html

```diff
@@ -36,7 +36,10 @@
                 return view.addLayer(layer);
             }
 
-             itowns.Fetcher.json('./layers/JSONLayers/OPENSM.json').then(addLayerCb);
+            itowns.Fetcher.json('./layers/JSONLayers/OPENSM.json').then(function _(config) {
+                var layer = new itowns.ColorLayer(config.id, config);
+                return view.addLayer(layer);
```

Comment:
with the `return`

Reply:
Because `addLayerCb` (which I forgot to remove) returned it, but there is no need indeed.